### DISCUSSION TITLE
Please tell me how to develop _tw more conveniently, refresh the browser, vscode, etc.

### DISCUSSION THREAD

**Discussion by aboutjquery** [gregsullivan/_tw#144] [General]
I uploaded _tw to wordpress on the dxxxxxxx host. Now after npm run watch, I need to refresh the browser manually every time, which is not convenient. Is there a good way?

In addition, I stupidly used winscp to connect to the host's _tw theme and modified the template to develop... Is there a way to use my vscode? How do you all like to develop? Install localwp? Or are there other methods? Thank you

**Comment by gregsullivan** [comment]
Hello—Thank for posting this! I've converted your issue into a discussion.

If you want live reloading, there is a documentation page on [setting up Browsersync](https://underscoretw.com/docs/browsersync/).

In the quickstart, I suggest using a local WordPress development environment, and I continue to think that's the best and most efficient approach. There are ways to use VS Code to develop on a server, but I wouldn't use them unless forced to by a project requirement of some kind. (I can't think of a specific requirement that would lead to that, but I allow that anything is possible!)

Hope that helps!
